src/hit/api/params.cpp

```diff
@@ -26,7 +26,7 @@ namespace hit {
           btp_params(optional<BootstrappingParams>(BootstrappingParams(move(lattigo_btp_params)))) {
     }
 
-    CKKSParams::CKKSParams(int num_slots, int log_scale, int max_ct_level, int num_ks_primes,
+    CKKSParams::CKKSParams(int num_slots, int max_ct_level, int log_scale, int num_ks_primes,
```

Comment:
Are we sure we've properly swapped these values in all possible callers? This feels like a dangerous refactor if we aren't super careful.

Reply:
I just triple-checked by adding a dummy parameter to this constructor and letting the compiler help me find all uses of this constructor. All of them are fixed in this PR.